### DISCUSSION TITLE
core function to append to a file

### DISCUSSION THREAD

**Discussion by shykes** [dagger/dagger#12137] [Ideas]
It would be nice to have a core function to append to an existing file.

**Comment by jedevc** [comment]
:tada: yeah, agreed :) It would be fairly easy to have this as a core api method, something like `File.append`.

**Comment by jedevc** [comment]
DagOp should help make this possible to implement when it lands: https://github.com/dagger/dagger/pull/9395

Since there's no LLB op to append to a file, we can't *currently* do this - but with DagOp, we'll just be able to implement it without that.
